modules/rest-util/src/blaze/middleware/output.clj

```diff
@@ -1,5 +1,7 @@
 (ns blaze.middleware.output
-  "JSON serialization middleware."
+  "Data serialization middleware.
```

Comment:
please leave it like it was

Reply:
"Non-FHIR JSON deserialization middleware."